Sources/CameraControlARView/RealityView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  RealityView.swift
+//  CompleteAnatomy
+//
+//  Created by Volodymyr Boichentsov on 11/10/2022.
+//  Copyright © 2022 3D4Medical, LLC. All rights reserved.
+//
+
+import SwiftUI
+import Combine
+import RealityKit
+
+fileprivate let arContainer = ARViewContainer.init(cameraARView: CameraControlARView.init(frame: .zero))
+var cancellables = [Cancellable]()
+
+public struct RealityKitView: View {
+    public typealias UpdateBlock = () -> Void
+    
+    public struct Context {
+        public var base: RealityKit.Scene?
+        
+        
+        public func add(_ entity:Entity) {
+            
+            let originAnchor = AnchorEntity(world: .zero)
+            originAnchor.addChild(entity)
+            arContainer.arView.scene.anchors.append(originAnchor)
+        }
+    }
+    let context = Context.init(base: arContainer.arView.scene)
+    var update: UpdateBlock?
+    
+    
+    public init(_ content: @escaping (_ context:Context) -> Void, update: UpdateBlock? = nil) {
+        content(context)
+        self.update = update
+        
+        if let update = self.update {
+            let updateCa = arContainer.arView.scene.subscribe(to: SceneEvents.Update.self) { event in
```

Comment:
can we name `updateCa` a more descriptive variable? I'm a little confused on this point - it makes sense to get the various SceneEvents as they're triggered, but you're not passing them down into the closure, so I'm not understanding the purpose of this all.

Reply:
I will rename `updateCa` to `updateCancellable` probably forget to finish typing :) 

Idea with update is that we can do something each frame. Update is optional argument.